### PR TITLE
Mentors

### PR DESCRIPTION
**Pull request by Zandario**
Porting the Mentor system we have on CitMain

### CODE REVIEW

File: code/modules/mentor/mentor.dm [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	owner = GLOB.directory[ckey]
+	if(owner)
+		owner.mentor_datum = src
+		owner.add_mentor_verbs()
+		if(!check_rights_for(owner, R_ADMIN,0)) // Don't add admins to mentor list.
+			GLOB.mentors += owner
+
+/datum/mentors/proc/CheckMentorHREF(href, href_list)
+	return TRUE
+	/*			Disabled
```
Comment (TheLordME) [inline code/modules/mentor/mentor.dm:35]: do you plan on implementing this later?

File: interface/stylesheet.dm [hunk 2]
```diff
@@ -59,6 +59,11 @@ em						{font-style: normal;font-weight: bold;}
 .admin_channel			{color: #9611D4;	font-weight: bold;}
 .event_channel			{color: #cc3399;	font-weight: bold;}
 
+//* Mentor *//
+.mentorsay				{color: #5E2D79;	font-weight: bold;}
```
Comment (TheLordME) [inline interface/stylesheet.dm:63]: indent fuckup?

**Review by TheLordME — APPROVED** [review]

File: code/modules/mentor/mentor.dm (hunk 1 above)
Comment (Zandario) [reply code/modules/mentor/mentor.dm:35]: CheckAdminHref does the same right now, this will be done in a different PR

**Comment by Shadow-Quill** [comment]
Currently, `mentor follow` is a verb visible to everyone in the right click menu on mobs. (Though clicking it does nothing.)

**Review by TheLordME — CHANGES_REQUESTED** [review]
Multitude of Fixes still required

**Review by silicons — COMMENTED** [review]
please check the verbs and make the names properly named for rementor/dementor
and make sure mentor_follow is not on every mob

also you should probably be granting the mentor follow/unfollow verbs when someone's like, a mentor, instead of it being on everyone

**Pull request closed without merging**